src/main/java/org/mule/service/scheduler/internal/DefaultSchedulerService.java

```diff
@@ -297,6 +304,22 @@ public SchedulerThreadPools load(SchedulerPoolsConfigFactory key) throws Excepti
 
       poolsMaintenanceScheduler = ioScheduler();
       poolsMaintenanceTask = poolsMaintenanceScheduler.scheduleAtFixedRate(() -> poolsByConfig.cleanUp(), 1, 1, MINUTES);
+
+      if (usageTraceIntervalSecs != null) {
+        usageReportingScheduler = cpuLightScheduler();
+
+        logger.info("Usage Trace enabled");
```

Comment:
I would use a dedicated logger category for this. So can can be enabled/disabled separately from other `DefaultSchedulerService` logging. It would also have side effect of reducing category noise in output.